values.yaml

```diff
@@ -1263,6 +1269,16 @@ ingressGateways:
   # case of annotations where both will be applied.
   gateways:
     - name: ingress-gateway
+      # Possibility to override default replicas per gateway
+      # replicas: 3
+      # Possibility to override default resources per gateway
+      # resources:
+      #   requests:
+      #     memory: "100Mi"
+      #     cpu: "100m"
+      #   limits:
+      #     memory: "500Mi"
+      #     cpu: "500m"
```

Comment:
This looks like it got here by accident 😃

Reply:
Not by accident @ishustava . I put on purpose because I'm sure some people won't dig into the `template` folder or `test/units` folder like I did :-) to see how it works.

Reply:
@ishustava anything blocking the merge?

Reply:
Hi, this is documented above (https://github.com/hashicorp/consul-helm/pull/609/files#diff-ba05023d0f1b5d4b64b6cad4cf9a7ecdR1160-R1167). I don't think it makes sense to document this here because we'd need to have a commented out config for all the possible settings which would be a lot of comments, this would also need to be done for `terminatingGateways` and finally we don't use commented-out config as documentation anywhere else in our chart so this also breaks that consistency.

Based on those reasons can you please remove this config.

Reply:
@lkysow thx for your feedback. I removed the extra comments.